### PR TITLE
Add addons together:

### PR DESCRIPTION
**Pull request by ivanredbread**
- for ClamAV
- for AWS label recognition
- for background removing

Use `fileinfo` to represent results of addon performance inside `appdata` block

## Checklist

- [ ] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

**Review by dmitry-mukhin — APPROVED** [review]
I'm not sure about API changes, code looks ok

**Review by genme — APPROVED** [review]

File: pyuploadcare/client.py [hunk 1]
```diff
@@ -146,6 +147,7 @@ def __init__(
         )
         self.webhooks_api = WebhooksAPI(client=self.rest_client, **api_config)  # type: ignore
         self.project_api = ProjectAPI(client=self.rest_client, **api_config)  # type: ignore
+        self.addons_api = AddonAPI(client=self.rest_client, **api_config)  # type: ignore
```
[the remark this answers is not in the document]
Comment (ivanredbread) [reply pyuploadcare/client.py]: You are right! `Addons` is more homogenous with other API-class naming using plural, fix it at other places too

File: tests/functional/api/test_addons.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+from uuid import uuid4
+
+import pytest
+
+from pyuploadcare.api.api import AddonAPI
+
+
+@pytest.fixture()
+def addon_api(uploadcare) -> Iterable[AddonAPI]:
+    yield uploadcare.addons_api
```
[the remark this answers is not in the document]
Comment (ivanredbread) [reply tests/functional/api/test_addons.py]: fix to `addons`

File: tests/functional/api/test_addons.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+from uuid import uuid4
+
+import pytest
+
+from pyuploadcare.api.api import AddonsAPI
+
+
+@pytest.fixture()
+def addons_api(uploadcare) -> Iterable[AddonsAPI]:
+    yield uploadcare.addons_api
```
Comment (homm) [inline tests/functional/api/test_addons.py:11]:
```suggestion
def addons_api(uploadcare) -> AddonsAPI:
    return uploadcare.addons_api
```

**Pull request merged**